src/main/java/org/folio/circulation/domain/RequestRepresentation.java

```diff
@@ -53,74 +40,61 @@ private static void addStoredItemProperties(JsonObject request, Item item) {
     request.put("item", itemSummary);
   }
 
-  private static void addStoredRequesterProperties(
-    JsonObject requestWithAdditionalInformation,
-    User requester) {
+  private static void addStoredRequesterProperties(JsonObject request, User requester) {
```

Comment:
Not a big deal, but since you're changing the name of the JsonObject parameter, might as well change it to something else so that later down in the file you are not forced to have the JsonObject be named "requestJO" instead of "request", and then use "request" for the parameter of type Request.

Reply:
I renamed it for short by type everybody can catch what it is, but yes only one place has a problem  with naming. Anyway I think local vars can be named whatever you want and independently with other methods. Just one condition it should be understandable 😄